examples/sr_get_items_example.c

```diff
@@ -131,13 +132,18 @@ main(int argc, char **argv)
             ds = SR_DS_RUNNING;
         } else if (!strcmp(argv[2], "operational")) {
             ds = SR_DS_OPERATIONAL;
+        } else if (!strcmp(argv[2], "startup")) {
+            ds = SR_DS_STARTUP;
+        } else if (!strcmp(argv[2], "candidate")) {
+            ds = SR_DS_CANDIDATE;
```

Comment:
Have not noticed this, please add the new datastores to the brief usage on line 126.

Reply:
Right, I missed that, thanks!